cfme/tests/services/test_config_provider_servicecatalogs.py

```diff
@@ -75,7 +78,15 @@ def test_order_tower_catalog_item(appliance, catalog_item, request, job_type):
         casecomponent: Services
         caseimportance: high
     """
-    service_catalogs = ServiceCatalogs(appliance, catalog_item.catalog, catalog_item.name)
+    if job_type == 'template_limit':
+        config_manager_obj = config_manager
```

Comment:
Q: Why not just use `config_manager` in L83 below? I.e. why is this variable assignment necessary?

Reply:
Done